test/mnist_1000.jl

```diff
@@ -0,0 +1,82 @@
+using LazySets, Test, LinearAlgebra, GLPKMathProgInterface
+using NeuralVerification
+import NeuralVerification: ReLU, Id
+using JLD2
```

Comment:
Do we need JLD2? It didn't look to me like anything was using it. If not, I'd rather remove the dependency.

Reply:
I stored 1000 mnist images in the file `mnist_1000.jld2`.
The file is loaded in `mnist_1000.jl` at line 19.

Reply:
Ah ok, and looks like you set the requirement as test-only, so it's fine by me.